tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "http://json.schemastore.org/tsconfig",
+  "compilerOptions": {
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "noImplicitAny": true,
+    "strictNullChecks": true,
+
+    "lib": ["es2017", "dom"],
```

Comment:
Please use `es2018` instead of `es2017`. Also, do you need `dom`? IIRC, we need `dom` lib in loopback-next because of a bug in `request` typings. Please check whether `dom` is really needed and remove it if it's not.